components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/java/org/wso2/carbon/identity/mgt/endpoint/client/api/NotificationApi.java

```diff
@@ -229,7 +217,64 @@ public void setPasswordPost(ResetPasswordRequest resetPasswordRequest) throws Ap
 
         String[] localVarAuthNames = new String[]{};
 
-
         apiClient.invokeAPI(localVarPath, "POST", localVarQueryParams, localVarPostBody, localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames, null);
     }
+
+    /**
+     * This API is used to validate code of self reigstered users
+     *
+     * @param code Code retried after user self registration and optional property parameters (required)
+     * @throws ApiException If fail to call the API, e.g. server error or cannot deserialize the response body
+     */
+    public void validateCodePostCall(CodeValidationRequest code) throws ApiException {
+
+        // verify the required parameter 'code' is set
+        if (code == null) {
+            throw new ApiException(400, "Missing the required parameter 'code' when calling validateCodePost(Async)");
+        }
+        List<Pair> queryParams = new ArrayList<>();
+        Map<String, String> headerParams = new HashMap<>();
+        Map<String, Object> formParams = new HashMap<>();
+        final String[] contentType = {
+                "application/json"
+        };
+        final String headerAccept = apiClient.selectHeaderAccept(contentType);
+        final String headerContentType = apiClient.selectHeaderContentType(contentType);
+        String tenantDomain = getTenantDomain(code);
+        setBasePath(tenantDomain);
+
+        if (headerAccept != null) {
+            headerParams.put("Accept", headerAccept);
+        }
+        headerParams.put("Content-Type", headerContentType);
```

Comment:
What is the reason to specifically check for null only for headerAccept.
Looks like both had been initiated over contentType variable defined above.

Reply:
Also, define the variable along with its usage so it improves readability as it is within the context being used

Reply:
headerAccept returns null if the contentType.length is 0. So we need to check null for headerAccept.  headerContentType returns a string if the contentType.length is 0. SO no need to check null here for headerContentType.